frontend/src/components/Footer/Footer.cy.js

```diff
@@ -0,0 +1,21 @@
+import { mount } from 'cypress/react18';
+import Footer from '.'
+
+describe('Footer', () => {
+  it('renders the footer with the right text and links', () => {
+    mount(<Footer />);
```

Comment:
```suggestion
    mount(<Footer />);

```

Só pra separar a pré-condição das verificações.